docs/md/executor.md

```diff
@@ -84,8 +84,14 @@ Example fetch of a method configuration's version from the appropriate
 Firecloud instance (prod):
 
 ```shell
-curl -X 'GET' \
-  'https://firecloud-orchestration.dsde-prod.broadinstitute.org/api/workspaces/emerge_prod/Arrays_test/method_configs/warp-pipelines/Arrays' \
+FIRECLOUD=https://firecloud-orchestration.dsde-prod.broadinstitute.org
+
+# Should be as they'd appear in a Terra UI URL path:
+WORKSPACE=emerge_prod/Arrays_test
+METHOD_CONFIG=warp-pipelines/Arrays
+
+curl -X GET \
+  "$FIRECLOUD/api/workspaces/$WORKSPACE/method_configs/$METHOD_CONFIG" \
```

Comment:
Thank you.
I think that conveys intent better than what was there before.

Reply:
I agree, thank you for planting the seed for this earlier.  I forget that not everyone has my background in Terra-land.